spec/project-spec.js

```diff
@@ -274,6 +274,51 @@ describe('Project', () => {
     })
   })
 
+  describe('atomProject', () => {
+    let newSettings, projectPath1, projectPath2
+    beforeEach(() => {
+      atom.project.replace(null)
+      projectPath1 = temp.mkdirSync('project-path1')
+      projectPath2 = temp.mkdirSync('project-path2')
+      newSettings = {
+        paths: [projectPath1, projectPath2],
+        originPath: "originPath",
+        config: {
+          "baz": "buzz"
+        }
+      }
```

Comment:
I'd like to avoid using the word 'settings' for this object, since there already is a very closely-related concept of 'project-specific settings' meaning just the config settings. Since this object also contains the paths, maybe we should call it something more general. How about 'project specification', 'project state', or just 'project info'?

@matthewwithanm any naming suggestions?